packages/preview/equate/0.1.0/assets/example.typ

```diff
@@ -0,0 +1,30 @@
+#import "../src/lib.typ": equate
```

Comment:
I did it this way because the example (used for the readme) should reflect the current state of the code, which may be different from the one in the `@preview` namespace. I can of course still change it if you want me to

Reply:
That makes sense, will do 👍